src/Http/Middleware/StartSession.php

```diff
@@ -70,7 +72,7 @@ private function makeSession(Request $request)
         $cookieName = $this->cookie->getName($this->config['cookie']);
 
         return new Store(
-            $cookieName,
```

Comment:
Can we let `$this->config['cookie']` be the value of `$cookieName` now? Alternatively, just inline it. ;)

Reply:
`$cookieName` is still used on line 77 though?

Reply:
Precisely. Either make it be used twice, or inline the temp var, please. :wink:

Reply:
Ah, gotcha :)